src/main/java/com/github/mauricioaniche/ck/CK.java

```diff
@@ -51,11 +50,18 @@ public CK() {
 	}
 
 	public void calculate(String path, CKNotifier notifier) {
-		String[] srcDirs = FileUtils.getAllDirs(path);
 		String[] javaFiles = FileUtils.getAllJavaFiles(path);
-		String[] allDependencies = useJars ? FileUtils.getAllJars(path) : null;
-
 		log.info("Found " + javaFiles.length + " java files");
+
+		calculate(path, notifier, javaFiles);
+	}
+
+	public void calculate(String environment, CKNotifier notifier, String... javaFiles) {
```

Comment:
Why is the variable called `environment`? Maybe use the same name in both `calculate` methods? One is called `path`. Just pick the one you prefer!

Reply:
I changed it to path :)